verilog/benchmarks_large/litex/soc_basesoc_minispartan6_lm32_ise.prj

```diff
@@ -1,4 +1,4 @@
-verilog work soc_basesoc_minispartan6_lm32.v
+verilog work soc_basesoc_minispartan6_lm32.v.gz
```

Comment:
Does this mean ISE can accept .gz files?

Reply:
Surely not. This got messed due the experiments with Litex/ISE. 
I've fixed it for lm32, rocket and vexriscv.